docs-v2/integrations/all/canny.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Canny
+sidebarTitle: Canny
+---
+
+API configuration: [`canny`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Features | Status |
+| - | - |
+| [Auth (API Key)](/integrate/guides/authorize-an-api) | ✅ |
+| [Sync data](/integrate/guides/sync-data-from-an-api) | ✅ |
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+
+-   [API Reference](https://developers.canny.io/api-reference)
+-   [Auyhentication](https://developers.canny.io/api-reference#authentication)
+
+<Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
+
+## API gotchas
+
+_None yet. Add yours!_
```

Comment:
I recently had a chat with a team member from Canny, and he confirmed to me that their endpoint also accepts any of the body data as url param, and also accepts `GET` requests for their endpoint. This is not mentioned in their documentation, it would be nice to have this in the API gotchas section.

Reply:
Very cool to know! I've added a note about that to the gotchas section. Thanks for the insight.